src/Sentry/SentryEvent.cs

```diff
@@ -286,6 +286,24 @@ public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger)
     /// </summary>
     public static SentryEvent FromJson(JsonElement json) => FromJson(json, null);
 
+    private static SentryLevel? SafeLevelFromJson(JsonElement json)
+    {
+        var levelString = json.GetPropertyOrNull("level")?.GetString();
+        if (levelString == null)
+            return null;
+
+        // Native SentryLevel.None does not exist in dotnet
```

Comment:
Do we know what the behaviour of SentryLevel.None is? Does it mean nothing gets logged? Is that was setting the level = null does?